### ISSUE TITLE
install.packages("sf") fails  to install sf 1.0-11 on ubuntu 20.04

### ISSUE THREAD

**Issue by jhnwllr** [r-spatial/sf#2125]
I am trying to install `sf` on a GitHub actions runner for testing. 

https://github.com/ropensci/rgbif/actions/runs/4446221632/jobs/7806276788

```
ℹ Building sf 1.0-11
✖ Failed to build sf 1.0-11
Error: 
! error in pak subprocess
Caused by error in `stop_task_build(state, worker)`:
! Failed to build source package 'sf'
Full installation output:
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
staged installation is only possible with locking
** using non-staged installation
configure: CC: gcc
configure: CXX: g++ -std=gnu++14
checking for gdal-config... /usr/bin/gdal-config
checking gdal-config usability... yes
configure: GDAL: 3.0.4
checking GDAL version >= 2.0.1... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for gdal.h... yes
checking GDAL: linking with --libs only... yes
checking GDAL: /usr/share/gdal/pcs.csv readable... no
checking GDAL: checking whether PROJ is available for linking:... yes
checking GDAL: checking whether PROJ is available fur running:... yes
configure: GDAL: 3.0.4
configure: pkg-config proj exists, will use it
configure: using proj.h.
configure: PROJ: 6.3.1
checking PROJ: checking whether PROJ and sqlite3 are available for linking:... yes
checking for geos-config... /usr/bin/geos-config
checking geos-config usability... yes
configure: GEOS: 3.8.0
checking GEOS version >= 3.4.0... yes
checking for geos_c.h... yes
checking geos: linking with -L/usr/lib/x86_64-linux-gnu -lgeos_c... yes
configure: Package CPP flags:   -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include
configure: Package LIBS: -lproj   -L/usr/lib -lgdal -L/usr/lib/x86_64-linux-gnu -lgeos_c
configure: creating ./config.status
config.status: creating src/Makevars
** libs
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c bbox.cpp -o bbox.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal.cpp -o gdal.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_geom.cpp -o gdal_geom.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_read.cpp -o gdal_read.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_utils.cpp -o gdal_utils.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_write.cpp -o gdal_write.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c geos.cpp -o geos.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c hex.cpp -o hex.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c mdim.cpp -o mdim.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c ops.cpp -o ops.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c polygonize.cpp -o polygonize.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c proj.cpp -o proj.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c proj_info.cpp -o proj_info.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c raster2sf.cpp -o raster2sf.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c sfc-sfg.cpp -o sfc-sfg.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c signed_area.cpp -o signed_area.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c stars.cpp -o stars.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c wkb.cpp -o wkb.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c zm_range.cpp -o zm_range.o
g++ -std=gnu++14 -shared -L/opt/R/4.2.3/lib/R/lib -L/usr/local/lib -o sf.so RcppExports.o bbox.o gdal.o gdal_geom.o gdal_read.o gdal_utils.o gdal_write.o geos.o hex.o mdim.o ops.o polygonize.o proj.o proj_info.o raster2sf.o sfc-sfg.o signed_area.o stars.o wkb.o zm_range.o -lproj -L/usr/lib -lgdal -L/usr/lib/x86_64-linux-gnu -lgeos_c -L/opt/R/4.2.3/lib/R/lib -lR
installing to /tmp/Rtmp21InZ5/pkg-lib2ca1[574](https://github.com/ropensci/rgbif/actions/runs/4446221632/jobs/7806276788#step:10:575)6aa0/sf/libs
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
in method for ‘dbWriteTable’ with signature ‘"PostgreSQLConnection","character","sf"’: no definition for class “PostgreSQLConnection”
in method for ‘dbDataType’ with signature ‘"PostgreSQLConnection","sf"’: no definition for class “PostgreSQLConnection”
in method for ‘coerce’ with signature ‘"Spatial","sf"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"Spatial","sfc"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"sf","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"sfc","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"XY","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"crs","CRS"’: no definition for class “CRS”
in method for ‘coerce’ with signature ‘"sgbp","sparseMatrix"’: no definition for class “sparseMatrix”
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
Error: Error: package or namespace load failed for ‘sf’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf/libs/sf.so':
  /tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf/libs/sf.so: undefined symbol: _Z21CPL_gdalmdimtranslateN4Rcpp6VectorILi16ENS_15PreserveStorageEEES2_S2_S2_S2_b
Error: Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf’
---
Backtrace:
1. pak::local_install_dev_deps()
2. pak:::remote(function(...) { …
3. err$throw(res$error)
---
Subprocess backtrace:
 1. base::withCallingHandlers(cli_message = function(msg) { …
 2. get("local_install_dev_deps_do_plan", asNamespace("pak"))(...)
 3. pak:::pkg_install_do_plan(proposal = NULL, lib = lib)
 4. pkgdepends::install_package_plan(plan = plan, lib = lib, num_workers = num…
 5. base::withCallingHandlers({ …
 6. pkgdepends:::handle_events(state, events)
 7. pkgdepends:::handle_event(state, i)
 8. pkgdepends:::stop_task(state, worker)
 9. pkgdepends:::stop_task_build(state, worker)
10. base::throw(new_pkg_build_error("Failed to build source package {pkg}", …
11. | base::signalCondition(cond)
12. global (function (e) …
Execution halted
Error: Process completed with exit code 1.ℹ Building sf 1.0-11
✖ Failed to build sf 1.0-11
Error: 
! error in pak subprocess
Caused by error in `stop_task_build(state, worker)`:
! Failed to build source package 'sf'
Full installation output:
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
staged installation is only possible with locking
** using non-staged installation
configure: CC: gcc
configure: CXX: g++ -std=gnu++14
checking for gdal-config... /usr/bin/gdal-config
checking gdal-config usability... yes
configure: GDAL: 3.0.4
checking GDAL version >= 2.0.1... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for gdal.h... yes
checking GDAL: linking with --libs only... yes
checking GDAL: /usr/share/gdal/pcs.csv readable... no
checking GDAL: checking whether PROJ is available for linking:... yes
checking GDAL: checking whether PROJ is available fur running:... yes
configure: GDAL: 3.0.4
configure: pkg-config proj exists, will use it
configure: using proj.h.
configure: PROJ: 6.3.1
checking PROJ: checking whether PROJ and sqlite3 are available for linking:... yes
checking for geos-config... /usr/bin/geos-config
checking geos-config usability... yes
configure: GEOS: 3.8.0
checking GEOS version >= 3.4.0... yes
checking for geos_c.h... yes
checking geos: linking with -L/usr/lib/x86_64-linux-gnu -lgeos_c... yes
configure: Package CPP flags:   -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include
configure: Package LIBS: -lproj   -L/usr/lib -lgdal -L/usr/lib/x86_64-linux-gnu -lgeos_c
configure: creating ./config.status
config.status: creating src/Makevars
** libs
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c bbox.cpp -o bbox.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal.cpp -o gdal.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_geom.cpp -o gdal_geom.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_read.cpp -o gdal_read.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_utils.cpp -o gdal_utils.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c gdal_write.cpp -o gdal_write.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c geos.cpp -o geos.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c hex.cpp -o hex.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c mdim.cpp -o mdim.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c ops.cpp -o ops.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c polygonize.cpp -o polygonize.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c proj.cpp -o proj.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c proj_info.cpp -o proj_info.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c raster2sf.cpp -o raster2sf.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c sfc-sfg.cpp -o sfc-sfg.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c signed_area.cpp -o signed_area.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c stars.cpp -o stars.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c wkb.cpp -o wkb.o
g++ -std=gnu++14 -I"/opt/R/4.2.3/lib/R/include" -DNDEBUG -DHAVE_PROJ_H -I/usr/include/gdal -I/usr/include -I'/home/runner/work/_temp/Library/Rcpp/include' -I/usr/local/include   -fpic  -g -O2  -Wall -pedantic -c zm_range.cpp -o zm_range.o
g++ -std=gnu++14 -shared -L/opt/R/4.2.3/lib/R/lib -L/usr/local/lib -o sf.so RcppExports.o bbox.o gdal.o gdal_geom.o gdal_read.o gdal_utils.o gdal_write.o geos.o hex.o mdim.o ops.o polygonize.o proj.o proj_info.o raster2sf.o sfc-sfg.o signed_area.o stars.o wkb.o zm_range.o -lproj -L/usr/lib -lgdal -L/usr/lib/x86_64-linux-gnu -lgeos_c -L/opt/R/4.2.3/lib/R/lib -lR
installing to /tmp/Rtmp21InZ5/pkg-lib2ca1[574](https://github.com/ropensci/rgbif/actions/runs/4446221632/jobs/7806276788#step:10:575)6aa0/sf/libs
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
in method for ‘dbWriteTable’ with signature ‘"PostgreSQLConnection","character","sf"’: no definition for class “PostgreSQLConnection”
in method for ‘dbDataType’ with signature ‘"PostgreSQLConnection","sf"’: no definition for class “PostgreSQLConnection”
in method for ‘coerce’ with signature ‘"Spatial","sf"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"Spatial","sfc"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"sf","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"sfc","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"XY","Spatial"’: no definition for class “Spatial”
in method for ‘coerce’ with signature ‘"crs","CRS"’: no definition for class “CRS”
in method for ‘coerce’ with signature ‘"sgbp","sparseMatrix"’: no definition for class “sparseMatrix”
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
Error: Error: package or namespace load failed for ‘sf’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf/libs/sf.so':
  /tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf/libs/sf.so: undefined symbol: _Z21CPL_gdalmdimtranslateN4Rcpp6VectorILi16ENS_15PreserveStorageEEES2_S2_S2_S2_b
Error: Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/tmp/Rtmp21InZ5/pkg-lib2ca15746aa0/sf’
---
Backtrace:
1. pak::local_install_dev_deps()
2. pak:::remote(function(...) { …
3. err$throw(res$error)
---
Subprocess backtrace:
 1. base::withCallingHandlers(cli_message = function(msg) { …
 2. get("local_install_dev_deps_do_plan", asNamespace("pak"))(...)
 3. pak:::pkg_install_do_plan(proposal = NULL, lib = lib)
 4. pkgdepends::install_package_plan(plan = plan, lib = lib, num_workers = num…
 5. base::withCallingHandlers({ …
 6. pkgdepends:::handle_events(state, events)
 7. pkgdepends:::handle_event(state, i)
 8. pkgdepends:::stop_task(state, worker)
 9. pkgdepends:::stop_task_build(state, worker)
10. base::throw(new_pkg_build_error("Failed to build source package {pkg}", …
11. | base::signalCondition(cond)
12. global (function (e) …
Execution halted
Error: Process completed with exit code 1.
```

**Comment by edzer** [comment]
I think this is a copy of #2121

**Comment by mverouden** [comment]
Resolved in Release 1.0-12!

**Comment by edzer** [comment]
@jhnwllr could you confirm this has been resolved?
